iosTraining-Reeen/APICaller.swift

```diff
@@ -9,11 +9,21 @@ import Foundation
 import YumemiWeather
 
 protocol WeatherServiceProtocol {
-    func getWeatherInformation() -> String
+    func getWeatherInformation()
+}
+
+protocol WeatherUpdateDelegate: AnyObject {
+    func didUpdateWeatherInformation(weatherInfo: String)
```

Comment:
[must]

- delegate メソッドの命名には、ある程度決まったパターンがあり、まずはそれに則って実装するといいです。
- delegate は状況報告です。
- 参考
	- [Swift Style Guide](https://google.github.io/swift/#delegate-methods) : Google の Swift Style Guild
	- [GitHub - raywenderlich/swift-style-guide: The official Swift style guide for raywenderlich.com.](https://github.com/raywenderlich/swift-style-guide#delegates)
		- [Recommendation on delegate methods naming convention? · Issue #175 · raywenderlich/swift-style-guide · GitHub](https://github.com/raywenderlich/swift-style-guide/issues/175)
	- [Delegation in Swift | Swift by Sundell](https://www.swiftbysundell.com/articles/delegation-in-swift/)

```suggestion
protocol WeatherServiceDelegate: AnyObject {
    func weatherService(didUpdateWeatherInformation weatherInfo: String)
```

Reply:
そちらの記事を参考に、`weatherConditionDidUpdate`という命名にします。